test/JDBC/expected/BABEL-2455.out

```diff
@@ -261,16 +261,8 @@ go
 -- trigger
 CREATE TRIGGER .tr2455 on t2455_base AFTER INSERT AS print 'triggered';
 go
-~~ERROR (Code: 33557097)~~
```

Comment:
Here, was it considering whole "dbo.tr2455" as trigger name?

Reply:
In T-SQL it is valid to specify leading dots, e.g.  `.myproc` or `..myproc` which are both equivalent to `myproc` . This form was one of the syntax varitions that was not handled previously and led to an error, but is handled correctly now (see PR summary at the top of this page).